packages/caspar-graphics/lib/caspar.js

```diff
@@ -159,6 +189,19 @@ export default class Caspar extends React.Component {
         <TransitionGroup component={FirstChild}>
           {shouldRender && <Template data={data} state={state} />}
         </TransitionGroup>
+
+        {referenceImage && (
+          <div
+            id="reference"
```

Comment:
Is this used?

Reply:
I use the id's as `comments`, just to orientate the developer, but I can take them out.

Reply:
You can leave it here, but I think React Dev Tools should cover most needs for it.